src/pages/new/index.js

```diff
@@ -172,7 +172,7 @@ export default () => (
     <ArticleSpotlightCard
       eyebrowText="Partnership Spotlight"
       title="Digital Equity Team of San Francisco"
-      description="Free Fiber Initiative summary we're working together to find new and different ways of building lasting infrastructure"
+      description="Free Fiber Inititiative summary we’re working together to find new and different ways of building lasting infrastructure."
```

Comment:
The original spelling of "Initiative" was correct. I see that you've copied this from the Figma design, but in this case, there's a typo there. Could you revert the spelling of that word back to the correct one?

Reply:
Yes, sorry!